### PR TITLE
Troubleshooting Ubuntu Dual Boot Page

### PR DESCRIPTION
**Pull request by sthirupa**
Team G Spring 2023 MRSD Wiki Entry Final - Troubleshooting Ubuntu Dual Boot

### CODE REVIEW

**Review by PallaviMadhukar — APPROVED** [review]

**Review by nevalsar — CHANGES_REQUESTED** [review]
Good start to the PR, but needs some work before merging.

Please re-raise the review request once changes are made.

File: wiki/computing/troubleshooting-ubuntu-dual-boot.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+date:
+  2023/04/30: null
+title: Ubuntu Dual Boot and Troubleshooting Guide
+published: true
+---
+This page serves as a tutorial and troubleshooting guide for dual booting Ubuntu alongside Windows for the uninitiated. Several difficulties can be encountered during setup if not aware of the process in advance. Read the following sections as needed and be aware of the potential issues brought up.
+
+> It is recommended to begin the dual boot process as soon as possible in case things go wrong so that project time is not wasted trying to reinstall Ubuntu for the fourth time, or wasting $300 on memory sticks that do not function with the laptop used for the dual boot, as the writers of this  page experienced.
```
[the remark this answers is not in the document]
Comment (sthirupa) [reply wiki/computing/troubleshooting-ubuntu-dual-boot.md]: Made the appropriate changes in latest commits.

File: wiki/computing/troubleshooting-ubuntu-dual-boot.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Create a bootable USB drive for Ubuntu
+First, acquire an empty USB pen drive that is 8 GB or larger and insert it into the computer of choice. Have at least 64 GB of unused hard disk space on the computer (only 25-30 GB is needed for installation but the rest is needed for packages and coursework). 
+
+Go to www.releases.ubuntu.com and select an LTS Release. Check with someone to know which version is trusted as the latest release may not work with all software needed. Download the .iso file for that release. Download and use balenaEtcher with that .iso and the inserted USB drive to create a bootable Ubuntu drive.
```
[the remark this answers is not in the document]
Comment (sthirupa) [reply wiki/computing/troubleshooting-ubuntu-dual-boot.md]: Made the appropriate changes in latest commits.

[the remark this answers is not in the document]
Comment (sthirupa) [reply wiki/computing/troubleshooting-ubuntu-dual-boot.md]: Made the appropriate changes in latest commits.

File: wiki/computing/troubleshooting-ubuntu-dual-boot.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Summary
+There are a few ways Ubuntu installation can go wrong or be delayed but this page hopefully will help a few people avoid major mistakes that held the writers of this page back a few weeks. After this guide the computer should be ready for installing browsers (such as Firefox), IDEs (such as VSCode or PyCharm), and libraries (such as mujoco or realsense-ros) as desired.
+
+## See Also:
+- [Ubuntu 14.04 on Chromebook](https://roboticsknowledgebase.com/wiki/computing/ubuntu-chromebook)
+- [Upgrading Ubuntu Kernels](https://roboticsknowledgebase.com/wiki/computing/upgrading-ubuntu-kenel)
+
+## Further Reading
```
[the remark this answers is not in the document]
Comment (sthirupa) [reply wiki/computing/troubleshooting-ubuntu-dual-boot.md:63]: Made the appropriate changes in latest commits.

File: wiki/computing/troubleshooting-ubuntu-dual-boot.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+## Move/combine partitions to make use of inaccessible partitions
+Partitions of unallocated data can only be incorporated into another partition using Extend Volume if the unallocated partition is to the right of an existing partition with an OS. If there are multiple partitions of unallocated data or it is in a place where it is not able to get extended, use gparted in Ubuntu. This can be done before installing Ubuntu by selecting “Try Ubuntu” when loading Ubuntu from the bootable USB drive. Open a terminal by pressing at the same time Ctrl+Alt+T and run the command “gparted”. Once the window opens, select the partition of unallocated space and click the “Resize/Move” option to move the partition to where it can be used by Extend Volume on the partition, or click a partition used for an OS and move the sides of the partition to occupy the desired amount of unallocated memory. After each desired operation, click the Resize/Move button to queue the operation.
```
[the remark this answers is not in the document]
Comment (sthirupa) [reply wiki/computing/troubleshooting-ubuntu-dual-boot.md]: Made the appropriate changes in latest commits.

File: wiki/computing/troubleshooting-ubuntu-dual-boot.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+Go to www.releases.ubuntu.com and select an LTS Release. Check with someone to know which version is trusted as the latest release may not work with all software needed. Download the .iso file for that release. Download and use balenaEtcher with that .iso and the inserted USB drive to create a bootable Ubuntu drive.
+
+## Create partitions safely
+Creating the Ubuntu partition on the hard drive for the dual boot can be done while installing Ubuntu from the USB drive but it is better to do it while in Windows Go to the Disk Management page in Windows by right clicking the Start menu. From there, right click on a partition in the middle labeled “NTFS” and click “Shrink Volume”. Do not shrink the EFI System Partition or the Recovery Partition on either end of the large partition. Type in the amount of MB to free up - this should be at least 65536 for anticipated future work. This amount in GB should appear to the right of the Windows partition shrank with the label “Unallocated”.
```
[the remark this answers is not in the document]
Comment (sthirupa) [reply wiki/computing/troubleshooting-ubuntu-dual-boot.md:15]: Made the appropriate changes in latest commits.

**Review by nevalsar — APPROVED** [review]
There is some scope to revise the text in this article, but the difficulty in writing a comprehensive troubleshooting guide for niche topics is acknowledged. I'm okay with merging this and leaving the task of improving this page to a separate PR.

**Pull request merged**